imports/ui/ranking/HongKongRanking.html

```diff
@@ -0,0 +1,7 @@
+<template name="HongKongRanking">
+	{{ > Ranking 
+		name="Hong Kong Mahjong ELO Rankings"
```

Comment:
Maybe this title and the Japanese title should be constants or assets.  Thoughts?